### PR TITLE
TDQ-1012, TDQ-1017: Softer content for Vendor Email Status Report, update the date format

### PR DESCRIPTION
**Pull request by czeczotka**

### CODE REVIEW

**Comment by czeczotka** [comment]
Mia, can you see this? @miaodon

**Comment by edjannoo** [comment]
Looks good to me

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqHeaderComplianceEmailNoHeaders.scala.html [hunk 1]
```diff
@@ -42,21 +46,31 @@ <h2>What you must do next</h2>
             Read the specification to learn how your application must submit the correct fraud prevention headers.
         </li>
         <li style="font-size: 19px; line-height: 1.315789474;">
-            Use the Test Fraud Prevention Headers API (found in the list of VAT (MTD) APIs on the API documentation page) to check that the fraud prevention headers submitted by your application meet the requirements of HMRC’s fraud prevention specification.
+            Use the Test Fraud Prevention Headers API (found in the list of VAT (MTD) APIs on the API documentation page) to check that the fraud prevention headers submitted by your application meet the requirements of HMRC's fraud prevention specification.
```
Comment (romankusnierz) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqHeaderComplianceEmailNoHeaders.scala.html:49]: I assume all other apostrophes are correct as well

Comment (czeczotka) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqHeaderComplianceEmailNoHeaders.scala.html:49]: We settled on standard ones '

**Pull request merged**